### ISSUE TITLE
Make Link model

### ISSUE THREAD

**Issue by egiurleo** [RagtagOpen/freefrom-map-backend#7]
This model will have the following fields:
- id (integer, primary key)
- criteria_id (integer, foreign key)
- state (string)
- text (string)
- url (string)

This model will have a multicolumn index on state and criteria_id. 

This work must include tests ensuring that you can create, modify, and delete models of this type.

This is currently blocked by #5.

**Comment by brittanylau** [comment]
Hey @egiurleo, this one looks like a duplicate of #8 – should we close this one?

**Comment by egiurleo** [comment]
Whoops! Yes! Thanks for catching that.
